packages/lib/src/vis/shared/PanZoomMesh.tsx

```diff
@@ -81,14 +77,12 @@ function PanZoomMesh() {
       // Prevent events from reaching tooltip mesh when panning
       evt.stopPropagation();
 
-      const projectedPoint = camera.worldToLocal(evt.unprojectedPoint.clone());
-      const { x: pointerX, y: pointerY } = projectedPoint;
-      const { x: startX, y: startY } = startOffsetPosition.current;
-
-      const targetX = startX - pointerX;
-      const targetY = startY - pointerY;
+      const delta = startOffsetPosition.current
+        .clone()
+        .sub(evt.unprojectedPoint);
+      const target = camera.position.clone().add(delta);
```

Comment:
Using world coordinates for `startOffsetPosition.current` makes possible to remove the projection here.

As a reminder, when panning, we want to compute the change in position from the starting position of the pointer to its current position (`delta`) and impact this change to the camera position (giving `target`).